static/js/components/dashboard/CourseDescription_test.js

```diff
@@ -105,15 +106,14 @@ describe('CourseDescription', () => {
       course.runs.length > 0 &&
       course.runs[0].status === STATUS_NOT_PASSED
     ));
-    alterFirstRun(course, {course_end_date: '1999-13-92'});
-    const wrapper = shallow(<CourseDescription course={course} />);
+    let firstRun = alterFirstRun(course, {course_end_date: '1999-13-92'});
+    const wrapper = shallow(<CourseDescription courseRun={firstRun} courseTitle={course.title} />);
     let elements = getElements(wrapper);
     assert.equal(elements.detailsText, '');
   });
 
   it('does not show anything when there are no runs for a course', () => {
-    let course = findCourse(course => course.runs.length === 0);
-    const wrapper = shallow(<CourseDescription course={course} />);
+    const wrapper = shallow(<CourseDescription courseRun={null} courseTitle={null} />);
```

Comment:
It looks like `CourseRow` passes an empty object to `CourseDescription` when there are no rows, not `null`, at least that's what I'm getting from `getFirstRun`. Can we make it consistent one way or the other?